openstack/networking/v2/extensions/vpnaas/services/requests.go

```diff
@@ -0,0 +1,41 @@
+package services
+
+import "github.com/gophercloud/gophercloud"
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToVPNServiceCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains all the values needed to create a new VPN service
+type CreateOpts struct {
```

Comment:
doc nit: all fields should have a descriptive comment even if it's something simple like:

```
Name is the human-readable name of the VPN Service.
```